my-app/src/components/to-do-list/ToDoList.css

```
@@ -0,0 +1,222 @@
+:root {
+  --bg: #f5f5f5;
+  --card: #ffffff;
+  --text: #111;
+  --muted: #888;
+
+  --primary: #007bff;
+  --chip: #eee;
+  --border: #ddd;
+
+  --radius: 10px;
+  --shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+
+  --space-xs: 0.2rem;
+  --space-sm: 0.4rem;
+  --space-md: 0.6rem;
+  --space-lg: 1rem;
+}
+
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+}
+body {
+  margin: 0;
+  background: var(--bg);
+  color: var(--text);
+  font-family: var(--font-family);
+}
+
+.to-do-list {
+  max-width: 520px;
+  margin: 2rem auto;
+  background: var(--card);
+  padding: var(--space-lg);
+  border-radius: var(--radius);
+  box-shadow: var(--shadow);
+}
+
+h1 {
+  margin: 0 0 var(--space-md);
+  text-align: center;
+  font-size: 1.35rem;
+}
+
+.to-do-list > div:first-of-type {
+  display: flex;
+  gap: var(--space-sm);
+  align-items: center;
+  margin-bottom: var(--space-md);
+}
+
+input[type="text"] {
+  flex: 1;
+  min-width: 0;
+  padding: var(--space-md);
+  border: 1px solid var(--border);
+  border-radius: var(--radius);
+  background: #fff;
+  color: var(--text);
+}
+
+button {
+  padding: var(--space-md) calc(var(--space-md) * 1.6);
+  border: none;
+  border-radius: var(--radius);
+  background: var(--chip);
+  cursor: pointer;
+}
+button:hover {
+  opacity: 0.9;
+}
+button:active {
+  transform: translateY(1px);
+}
+
+.filters {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--space-sm);
+  justify-content: center;
+  margin: var(--space-md) 0 var(--space-lg);
+}
+.filter-btn {
+  background: var(--chip);
+}
+.filter-btn.active {
+  background: var(--primary);
+  color: #fff;
+}
+
+.task-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.task-list li {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  gap: var(--space-sm);
+  padding: var(--space-sm) 0;
+  border-bottom: 1px solid var(--border);
+}
+.task-list li:last-child {
+  border-bottom: 0;
+}
+
+.task-list li > button:first-child {
+  min-width: 2rem;
+  min-height: 2rem;
+  border-radius: 8px;
+  background: var(--chip);
+}
+
+.text {
+  margin: 0 var(--space-sm);
+  user-select: none;
+}
+.task-list li.completed .text {
```

be attentive to the formatting
```suggestion

.task-list li.completed .text {
```